background.js

```diff
@@ -136,16 +156,17 @@ function submitVote(type, UUID, callback) {
 function submitTimes(videoID, callback) {
   //get the video times from storage
   let sponsorTimeKey = 'sponsorTimes' + videoID;
-  chrome.storage.sync.get([sponsorTimeKey], function(result) {
+  chrome.storage.sync.get([sponsorTimeKey, "serverAddress"], function(result) {
```

Comment:
You forgot to add userID here, so `undefined` was submitted instead.

Reply:
I think we'd benefit from some test cases :)